securedrop_client/api_jobs/downloads.py

```diff
@@ -375,5 +375,5 @@ def call_decrypt(self, filepath: str, session: Session = None) -> str:
             filepath, plaintext_filepath, is_doc=True
         )
         logger.info("Decrypted file '{}' to folder '{}'".format(
-            filepath, os.path.dirname(original_filename)))
+            os.path.basename(filepath), os.path.dirname(filepath)))
```

Comment:
Note that this use of `os.path.basename` fixes another bug; the folder would otherwise be logged twice.